woocommerce/woocommerce-admin
# Remove fragile selector.

Fixes #7052 

There is some source I haven't identified that adds a class `nosub` to the element with class `wrap`. this causes the fallback selector in this script to fail (it looks for an element with *just* the class wrap and no other classes) and therefore the wrap is null, and an exception is thrown trying to find the next sibling.

To fix the issue I've queried by the element having *at least* the class `wrap` which is less fragile, but quite honestly its still not ideal. This code should not be relying on having such a fragile and ambiguous html structure in place, but maybe thats something we can fix later.
### Accessibility

n/a

### Screenshots

n/a 

### Detailed test instructions:

-  Open the Products > Tags page
-  Check there are no console errors for `nextSibling`.

(Note I have noticed another unrelated console error, but I'll look to tackle that in a separate PR).

No changelog required (it seems this was cherry picked)
